Tutorials/SDF/car_model.sdf

```diff
@@ -0,0 +1,186 @@
+            <collision name='collision'>
+                <geometry>
+                    <cylinder>
+                        <radius>0.3</radius>
+                        <length>0.2</length>
+                    </cylinder>
+                </geometry>
+            </collision>
+        </link>   
+        <!--copy and paste for right wheel but change position-->
+        <link name='right_wheel'> <!--todo: change the inertia matrix-->
+            <pose relative_to="chassis">-0.5 -0.6 0 -1.5707 0 0</pose> <!--angles are in radian-->
+            <inertial>
+                <mass>2</mass>
+                <inertia>
+                    <ixx>0.145833</ixx>
+                    <ixy>0</ixy>
+                    <ixz>0</ixz>
+                    <iyy>0.145833</iyy>
+                    <iyz>0</iyz>
+                    <izz>0.125</izz>
+                </inertia>
+            </inertial>
+            <visual name='visual'>
+                <geometry>
+                    <cylinder>
+                        <radius>0.4</radius>
+                        <length>0.2</length>
+                    </cylinder>
+                </geometry>
+                <material>
+                    <ambient>1.0 0.0 0.0 1</ambient>
+                    <diffuse>1.0 0.0 0.0 1</diffuse>
+                    <specular>1.0 0.0 0.0 1</specular>
+                </material>
+            </visual>
+            <collision name='collision'>
+                <geometry>
+                    <cylinder>
+                        <radius>0.4</radius>
+                        <length>0.2</length>
+                    </cylinder>
+                </geometry>
+            </collision>
+
+        </link>
+
+        <!--caster wheel-->
+        <link name='caster'>
+            <pose relative_to='F'/> <!--todo: define a frame and use in the visual and collision-->
+            <inertial>
+                <mass>1</mass>
+                <inertia>
+                    <ixx>0.1</ixx>
+                    <ixy>0</ixy>
+                    <ixz>0</ixz>
+                    <iyy>0.1</iyy>
+                    <iyz>0</iyz>
+                    <izz>0.1</izz>
+                </inertia>
+            </inertial>
+            <visual name='visual'>
+                <geometry>
+                    <sphere>
+                        <radius>0.2</radius>
+                    </sphere>
+                </geometry>
+                <material>
+                    <ambient>0.0 1 0.0 1</ambient>
+                    <diffuse>0.0 1 0.0 1 1</diffuse>
+                    <specular>0.0 1 0.0 1</specular>
+                </material>
+            </visual>
+            <collision name='collision'>
+                <geometry>
+                    <sphere>
+                        <radius>0.2</radius>
+                    </sphere>
+                </geometry>
+            </collision>
+        </link>
+
+        <frame name="F" attached_to='chassis'>
+            <pose>0.8 0 -0.45 0 0 0</pose>
```

Comment:
With this current offset, the caster wheel is outside the chassis, but because it's so large, it makes the model tilt back:

![before_high_caster](https://user-images.githubusercontent.com/5751272/83780660-4f7f0c00-a642-11ea-95fd-d1acac5df691.png)

How about changing the offset to match the caster's radius, so the vehicle lies parallel to the ground?

```suggestion
            <pose>0.8 0 -0.2 0 0 0</pose>
```

![caster_after](https://user-images.githubusercontent.com/5751272/83780786-7b01f680-a642-11ea-83e8-a212fb74dcd1.png)

An alternative is to make the caster's radius smaller.